src/ewatercycle/models/wflow.py

```diff
@@ -41,7 +43,7 @@ def __init__(
         parameter_set: ParameterSet,
         forcing: Optional[WflowForcing] = None,
     ):
-
+        """Instantiate Wflow model object."""
```

Comment:
Same as for pcrglob, please remove this doc string.

Reply:
Done. Notice that the linter complained about this, so I had to add noqa flags to silence the linter.